packages/bot/package.json

```
@@ -13,19 +13,19 @@
     ]
   },
   "scripts": {
-    "start": "eslint ./src && rimraf dist &&  webpack-dev-server --mode=development --open 'Google Chrome' --hot --inline --port 80",
-    "serve": "webpack --mode=development --progress --watch --open 'Google Chrome' --hot --inline --port 80",
-    "build": "rimraf dist && cross-env NODE_ENV=production webpack --mode=production --optimize-minimize --progress",
-    "build:travis": "echo \"Error: no build:travis specified\"",
+    "start": "rimraf dist && npm run test && webpack --mode=development --progress --watch",
+    "serve": "rimraf dist && webpack --mode=development --progress --watch",
+    "build": "rimraf dist && npm run test && cross-env NODE_ENV=production webpack --mode=production --optimize-minimize --progress",
+    "build:travis": "echo \"No build:travis specified\"",
     "test": "eslint ./src && jest -w 1 --coverage",
     "test:eslint": "eslint ./src",
     "test:stylelint": "stylelint \"./src/**/*.s?(a|c)ss\"",
-    "test:mocha": "jest -w 1 --coverage",
-    "deploy": "gh-pages --dist '.' --src '{index.html,manifest.json,dist/**,CNAME}'",
-    "deploy:clean": "echo \"Error: no deploy:staging specified\"",
-    "deploy:folder": "gh-pages --dist '.' --src '{index.html,manifest.json,dist/**}' --dest",
-    "deploy:staging": "d() { test -z $1 && echo 'Please specify branch.' && exit 1; (git show-branch $1) || exit 1;  git stash; git checkout $1; npm i; rm -rf branch/$1; mkdir -p branch/$1 ; gulp build-min; cp -r www/ branch/$1; gulp release-branch --branch $1;}; d",
-    "deploy:production": "d() { git stash; git checkout master; npm i;gulp build-min;gulp release-master;}; d"
+    "test:mocha": "jest -w 10 --coverage",
```

Keep this on 1 worker or tests will be failing again due to making too many requests to API.

```suggestion
    "test:mocha": "jest -w 10 --coverage",
```

---

@aaron-binary you mean 10 workers?